nltepop.cc

```diff
@@ -473,24 +484,24 @@ void nltepop_matrix_add_boundbound(const int nonemptymgi, const int element, con
 
 void nltepop_matrix_add_ionisation(const int nonemptymgi, const int element, const int ion,
                                    const std::vector<double> &s_renorm, gsl_matrix *rate_matrix_rad_bf,
-                                   gsl_matrix *rate_matrix_coll_bf) {
-  assert_always((ion + 1) < get_nions(element));  // can't ionise the top ion
+                                   gsl_matrix *rate_matrix_coll_bf, const int first_ion_used, const int nions_used) {
+  assert_always((ion + 1) < nions_used + first_ion_used);  // updated to cover case when ions have been stripped
```

Comment:
< takes precedence over +, so I think there is a bug here. The old comment "can't ionise the top ion" made a lot more sense to me as a hint about what the condition is actually testing. The history of changes should be in git, not code comments.